services/src/app.js

```diff
@@ -71,13 +74,19 @@ app.configure(channels);
 // Configure a middleware for 404s and the error handler
 app.use(express.notFound());
 
-/* Avoid cluttering the test logs with expected errors and exceptions */
-if (process.env.NODE_ENV != 'test') {
-  app.use(express.errorHandler({ logger }));
-}
-else {
-  app.use(express.errorHandler());
-}
+/* eslint-disable no-unused-vars */
+app.use((err, req, res, next) => { 
```

Comment:
AIUI, re-reading https://expressjs.com/en/guide/error-handling.html, we are missing the block:

```js
if (res.headersSent) {
  return next(err);
}
```

And if you agree, that would allow us to get rid of the eslint-disable above.

Reply:
I was going back and forth on this, I can add this back in.